README.md

```diff
@@ -276,7 +276,7 @@ Then you can run the demo script using:
 ```bash
     mkdir -p docker-volume && docker run -v $PWD/docker-volume:/opt/Desktop \
     -e DISPLAY=$DISPLAY -v /tmp/.X11-unix:/tmp/.X11-unix --shm-size=2g \
-    -it openwpm python3 /opt/OpenWPM/demo.py
```

Comment:
This change is already in the current master. Can you update your master?

Reply:
you are right. thats on me.